src/Azs.Backup.Admin/test/Restore-AzsBackup.Tests.ps1

```diff
@@ -1,8 +1,8 @@
-$loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
-if (-Not (Test-Path -Path $loadEnvPath)) {
-    $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
-}
-. ($loadEnvPath)
+Remove-Module -Name "Az.Accounts"
```

Comment:
@TheOnlyWei did you try removing this import-module and run the test again after your fix to loadenvjson.ps1 ?  I think import-module az.accounts is not required here and other tests in this module.

Reply:
I removed it, here is the new build:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=437445&view=logs&j=6399d89d-3d30-5fff-3e92-07bbaf7ebe2f&t=6399d89d-3d30-5fff-3e92-07bbaf7ebe2f